src/Idris/IDEMode/REPL.idr

```diff
@@ -199,6 +199,10 @@ process (MakeWith l n)
     = replWrap $ Idris.REPL.process (Editing (MakeWith False (fromInteger l) (UN $ mkUserName n)))
 process (DocsFor n modeOpt)
     = replWrap $ Idris.REPL.process (Doc $ APTerm (PRef EmptyFC (UN $ mkUserName n)))
+process (DocsForTypeOf n modeOpt)
+    = do (REPL (TermChecked itm ity)) <- Idris.REPL.process $ (Check $ PRef replFC (UN $ mkUserName n))
+           | err => pure $ REPL $ REPLError (pretty0 $ show err)
+         process (DocsFor (prettyBy Syntax ity) modeOpt)
```

Comment:
I have no idea how to work with ity properly in this place. Convert `ity : IPTerm` to `PTerm` somehow?